network/src/peers/peer_info.rs

```diff
@@ -49,7 +52,7 @@ pub struct PeerQuality {
     pub rtt_ms: AtomicU64,
     /// The number of failures associated with the peer; grounds for dismissal.
     pub failures: AtomicU32,
-    /// The number of remaining blocs to sync with.
+    /// The number of remaining blocks to sync with.
     pub remaining_sync_blocks: AtomicU16,
```

Comment:
note: changing this to `AtomicU32` will fix https://github.com/AleoHQ/snarkOS/issues/716